rules_ruby_gapic/BUILD.bazel

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Executable targets defining the various flavors of gapic-generator ruby
+as a 'ruby binaries' (actually shellscripts) for use as a protoc plugin
+"""
+load ("//rules_ruby_gapic:ruby_binary/ruby_binary.bzl", "ruby_binary")
+
+##
+# Creates a bazel-runnable shellscript wrapping gapic-generator-ruby vanilla
+# ready to be uses as a protoc plugin
+#
+ruby_binary (
+  name = "gapic_generator_ruby",
+  entrypoint = "@gapic_generator_src//:gen_src/gapic-generator/bin/protoc-gen-ruby_gapic",
```

Comment:
there are three rules becase because gapic-generator-ruby itself has 3 entrypoints. the vanilla entrypoint is mainly for testing but I don't see a good reason for excluding it.